tests/basic_/models.py

```diff
@@ -6,3 +6,8 @@ class Author(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class Book(models.Model):
```

Comment:
I renamed to Blob so that if we later add a Book with the usual foreign key to Author, it won't have a JSONField appendage. :-)